databricks/koalas/series.py

```diff
@@ -2282,15 +2292,44 @@ def apply(self, func, args=(), **kwds):
         New York    3.044522
         Helsinki    2.484907
         Name: 0, dtype: float64
+
+
+        You can omit the type hint and let Koalas infer its type.
+
+        >>> s.apply(np.log)
+        London      2.995732
+        New York    3.044522
+        Helsinki    2.484907
+        Name: 0, dtype: float64
+
         """
-        assert callable(func), "the first argument should be a callable function."
-        spec = inspect.getfullargspec(func)
-        return_sig = spec.annotations.get("return", None)
-        if return_sig is None:
-            raise ValueError("Given function must have return type hint; however, not found.")
+        assert callable(func) or hasattr(func, "__call__"), \
+            "the first argument should be a callable function."
```

Comment:
`np.log` failed `callable(...)` check ..

Reply:
FYI: seems working in my local:

```py
>>> import numpy as np
>>> np.__version__
'1.16.4'
>>> callable(np.log)
True
```

Reply:
Mine was older and the error message was misreading in my case ... let me fix it. seems it returns `True` in my case too anyway.